src/mscorlib/src/System/Text/StringBuilder.cs

```diff
@@ -1308,7 +1308,10 @@ void ISerializable.GetObjectData(SerializationInfo info, StreamingContext contex
         private static void FormatError() {
             throw new FormatException(Environment.GetResourceString("Format_InvalidString"));
         }
-        
+
+        public static readonly int Index_Limit = 1000000;
+        public static readonly int Width_Limit = 1000000;
```

Comment:
We do not add new public APIs without going through the [API review process](https://github.com/dotnet/corefx/blob/master/Documentation/project-docs/api-review-process.md)